bundles/org.openhab.binding.sensorcommunity/src/test/java/org/openhab/binding/sensorcommunity/internal/NumberTest.java

```diff
@@ -58,4 +63,18 @@ public void testStringNumbers() {
         // System.out.println("D1R1 " + d1r1);
         assertEquals("1.9", Double.toString(d1r1), "Double 1.94, 1 place");
     }
+
+    @Test
+    public void testDecibelPattern() {
+        String d1 = "1.94 dB";
+        QuantityType<Dimensionless> dbQt = (QuantityType<Dimensionless>) QuantityType.valueOf(d1);
+        assertEquals(Units.DECIBEL, dbQt.getUnit());
+    }
+
+    @Test
+    public void testMicrogramQubicMeterPattern() {
+        String d1 = "1.94 µg/m³";
+        QuantityType<Density> pressureQt = (QuantityType<Density>) QuantityType.valueOf(d1);
+        assertEquals(Units.MICROGRAM_PER_CUBICMETRE, pressureQt.getUnit());
+    }
```

Comment:
These tests are not testing anything in the binding, but rather core UoM logic, so would belong to core (if they are not already there).